dex-operator/README.md

```diff
@@ -0,0 +1,48 @@
+# Dex Operator
+
```

Comment:
Could you please add a small description to what Dex is and insert a link to the project